<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>256</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>424</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_forcewindow">
   <property name="geometry">
    <rect>
     <x>-190</x>
     <y>0</y>
     <width>811</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_forcewindow{
    border: 2px solid black;
    border-radius: 4px;
    padding: 4px;
}
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Force Setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>421</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QLabel" name="label_force_laser">
      <property name="maximumSize">
       <size>
        <width>291</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Developing...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_force_laser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#button_force_laser{
    background-color: rgb(77, 155, 232);
}
</string>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="status_force_laser">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#status_force_laser {
    background-color: rgb(255, 0, 0);
    border-radius: 15px;
}
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>421</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_force_oil">
      <property name="maximumSize">
       <size>
        <width>291</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Developing...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_force_oil">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#button_force_oil{
    background-color: rgb(77, 155, 232);
}
</string>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="status_force_oil">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#status_force_oil{
    background-color: rgb(255, 0, 0);
    border-radius: 15px;
}
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_force_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_force_cancel{
    background-color: rgb(211, 211, 211);
}
</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
